L/libcups/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcups"
+version = v"2.3.3"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/michaelrsweet/libcups.git", "1fbe09e7a69774b2be4e10e12f13f8f3bf751f59")
```

Comment:
https://github.com/openprinting/cups looks a more "official" repo (and it's what's linked in https://openprinting.github.io/cups/)

Reply:
```suggestion
    GitSource("https://github.com/openprinting/cups.git", "12ff481989924fc82ffdf1b8699753a486a33547")
```